js/utils/constants.js

```diff
@@ -859,9 +1133,151 @@
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_HLS
+         * @type {string}
+         */
+        WIDEVINE_HLS: "wv_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.WIDEVINE_WVM ('wv_wvm')</code>:
+         *   An encoding type for widevine wvm streams.
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_WVM
+         * @type {string}
+         */
+        WIDEVINE_WVM: "wv_wvm",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.WIDEVINE_MP4 ('wv_mp4')</code>:
+         *   An encoding type for widevine mp4 streams.
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_MP4
+         * @type {string}
+         */
+        WIDEVINE_MP4: "wv_mp4",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.RTMP ('rtmp')</code>:
+         *   An encoding type for non-drm rtmp streams.
+         * @constant OO.VIDEO.ENCODING.RTMP
+         * @type {string}
+         */
+        RTMP: "rtmp",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AUDIO ('audio')</code>:
+         *   An encoding type for non-drm audio streams.
+         * @constant OO.VIDEO.ENCODING.AUDIO
+         * @type {string}
+         */
+        AUDIO: "audio",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.SMOOTH ('smooth')</code>:
+         *   An encoding type for non-drm smooth streams.
+         * @constant OO.VIDEO.ENCODING.SMOOTH
+         * @type {string}
+         */
+        SMOOTH: "smooth",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.FAXS_HLS ('faxs_hls')</code>:
+         *   An encoding type for adobe faxs streams.
+         * @constant OO.VIDEO.ENCODING.FAXS_HLS
+         * @type {string}
+         */
+        FAXS_HLS: "faxs_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD_VOD ('akamai_hd_vod')</code>:
+         *   An encoding type for akamai hd vod streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD_VOD
+         * @type {string}
+         */
+        AKAMAI_HD_VOD: "akamai_hd_vod",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HLS ('akamai_hd2_vod_hls')</code>:
+         *   An encoding type for akamai hd2 vod hls streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HLS
+         * @type {string}
+         */
+        AKAMAI_HD2_VOD_HLS: "akamai_hd2_vod_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HDS ('akamai_hd2_vod_hds')</code>:
+         *   An encoding type for akamai hd2 vod hds streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HDS
+         * @type {string}
+         */
+        AKAMAI_HD2_VOD_HDS: "akamai_hd2_vod_hds",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.IMA ('ima')</code>:
+         *   A string that repsents a video stream that is controlled and configured directly by IMA.
+         * @constant OO.VIDEO.ENCODING.IMA
+         * @type {string}
+         */
+        IMA: "ima"
+      }
+
+    };
+
```

Comment:
Assorted is any encryption that don't fall into a group.  Smooth may be common but other assorted are not so I was just keeping the miscenaleous ones grouped so that they would be easy to find in the list.  Do you want me to put the assorted group before widevine or break it up?